pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -151,6 +151,7 @@ func addPodReconciler(mgr kube_ctrl.Manager, rt core_runtime.Runtime, converter
 		PodConverter: k8s_controllers.PodConverter{
 			ServiceGetter:     mgr.GetClient(),
 			NodeGetter:        mgr.GetClient(),
+			GatewayGetter:     mgr.GetClient(),
```

Comment:
Not clear why we need the client 3 times in here. Maybe clean this up in a follow-up PR?

Reply:
Removed it since we're not looking up Gateways anymore